config/docker_compose.toml

```diff
@@ -318,6 +319,7 @@ cybersource = { payment_method = "card" }
 nmi = { payment_method = "card" }
 payme = { payment_method = "card" }
 deutschebank = { payment_method = "bank_debit" }
+paybox = { payment_method = "card" }
```

Comment:
Please add these URL's in `loadtest/config/development.toml` as well

Reply:
added